File: kotlin/prime.kt
fun isPrime(n: Int): Boolean {
  for (i in 2 until n) {
    if (n.rem(i) == 0) {
      return false
    }
  }
  return n > 1
}

fun main() {
  var i = 1
  var found = 0
  while (found < 3333) {
    if (isPrime(i)) {
      found++
    }
    i++
  }
}

File: go/prime.go
package main

func isPrime(n int) bool {
	for i := 2; i < n; i++ {
		if n%i == 0 {
			return false
		}
	}
	return n > 1
}

func main() {
	for i, found := 1, 0; found < 3333; i++ {
		if isPrime(i) {
			found++
		}
	}
}
